bears/c_languages/ClangComplexityBear.py

```diff
@@ -13,6 +14,7 @@ class ClangComplexityBear(LocalBear):
     """
 
     LANGUAGES = ClangBear.LANGUAGES
+    REQUIREMENTS = {PipRequirement('libclang-py3', '0.2')}
```

Comment:
just use ClangBear.REQUIREMENTS at all other Clang related bears to avoid duplication